frontend/pages/admin/IntegrationsPage/cards/ConditionalAccess/components/OktaConditionalAccessModal/OktaConditionalAccessModal.tsx

```diff
@@ -0,0 +1,396 @@
+import React, { useCallback, useContext, useEffect, useState } from "react";
+import { size } from "lodash";
+
+import { NotificationContext } from "context/notification";
+import { AppContext } from "context/app";
+import configAPI from "services/entities/config";
+import conditionalAccessAPI from "services/entities/conditional_access";
+import { IConfig } from "interfaces/config";
+import endpoints from "utilities/endpoints";
+
+// @ts-ignore
+import InputField from "components/forms/fields/InputField";
+import CustomLink from "components/CustomLink";
+import Modal from "components/Modal";
+import Button from "components/buttons/Button";
+import Icon from "components/Icon";
+import TooltipWrapper from "components/TooltipWrapper";
+import { IInputFieldParseTarget } from "interfaces/form_field";
+import { getErrorReason } from "interfaces/errors";
+import { LEARN_MORE_ABOUT_BASE_LINK } from "utilities/constants";
+import FileUploader from "components/FileUploader";
+
+const baseClass = "okta-conditional-access-modal";
+
+const OKTA_IDP_ID = "okta_idp_id";
+const OKTA_ACS_URL = "okta_assertion_consumer_service_url";
+const OKTA_AUDIENCE_URI = "okta_audience_uri";
+const OKTA_CERTIFICATE = "okta_certificate";
```

Comment:
Nit, Why not just have these form field names be the abbreviated variable names? They are explicitly set
to the appropriate JSON fields in the submission handler anyway.